lib/dkdeploy/typo3/cms/helpers/mysql.rb

```diff
@@ -0,0 +1,26 @@
+module Dkdeploy
+  module Typo3
+    module Cms
+      module Helpers
+        # MySQL Helpers
+        module Mysql
+          # Escape special character in string for MySQL.
+          # copied from https://github.com/tmtm/ruby-mysql/blob/2.9.14/lib/mysql.rb#L56
+          # @param [String]
+          # @return [String]
+          def mysql_escape_string(str)
+            str.gsub(/[\0\n\r\\\'\"\x1a]/) do |s|
```

Comment:
Please use readable variable names.
See http://bensmith.io/20-tips-for-better-naming